### PR TITLE
Add .gitattributes that were not initially included 

### PR DESCRIPTION
**Pull request by zhang-alvin**
Running tests can lead to unintended behavior with some comparison files because the .gitattributes files were not added.

### CODE REVIEW

File: proteus/tests/cylinder2D/ibm_method/mesh.edge [hunk 1]
```diff
@@ -1,3 +1,335 @@
-version https://git-lfs.github.com/spec/v1
-oid sha256:3f9c96cdbab38fd8e3a2ee647c41a63731f836028382b1b692460530757c1680
-size 5840
+333  1
```
Comment (cekees) [inline proteus/tests/cylinder2D/ibm_method/mesh.edge]: Why put this one under git? Just curious.

Comment (zhang-alvin) [reply proteus/tests/cylinder2D/ibm_method/mesh.edge]: Actually, this file was inadvertently added. It's not supposed to be here.

Comment (zhang-alvin) [reply proteus/tests/cylinder2D/ibm_method/mesh.edge]: It was the same error you ran into in which git auto added the file as one that needed to be committed.  

Force pushed same commit but omitting mesh.edge

**Comment by zhang-alvin** [comment]
`git status` with hashdist build gives: 

```
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

	modified:   proteus/tests/solver_tests_mprans/comparison_files/comparison_twp_navier_stokes_cavity_2d_p_t1.csv

Untracked files:
```

`comparison_twp_navier_stokes_cavity_2d_p_t1.csv` is not tracked by git-lfs so there are no other files not being accounted for by missing `.gitattributes` files.

**Pull request merged**